extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -456,6 +456,9 @@ export class AmpConsent extends AMP.BaseElement {
           return Promise.resolve(consentRequired);
         }
         return this.getConsentRemote_().then(consentResponse => {
+          if (!consentResponse) {
```

Comment:
would response ever be null?  I think we've verified that `checkConsentHref` is specified if `consentRequired` is not `true/false`. 

Still good to protect here though.

Reply:
Ahh yes you're right, https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/0.1/consent-config.js#L222-L233

Probably unnecessary to check guard here then right?

Reply:
I think protecting against `null` `undefined` is always good to have!